vendor/code.cloudfoundry.org/cli/api/cloudcontroller/ccv3/internal/api_routes.go

```diff
@@ -103,6 +103,7 @@ const (
 	GetSpaceStagingSecurityGroupsRequest                        = "GetSpaceStagingSecurityGroups"
 	GetSSHEnabled                                               = "GetSSHEnabled"
 	GetStacksRequest                                            = "GetStacks"
+	GetTasksRequest                                             = "GetTasks"
```

Comment:
As mentioned in your PR, we should indeed wait for https://github.com/cloudfoundry/cli/pull/2343 to be merged. I make this comment to hold the merge until the PR can properly bumbs github.com/cloudfoundry/cli